rollup.config.js

```diff
@@ -0,0 +1,57 @@
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import babelrc from 'babelrc-rollup'
+import pkg from './package.json'
+
+let external = Object.keys(pkg.dependencies)
+
+export default [
+  // browser-friendly UMD build
+  {
+    input: 'src/index.js',
+    output: {
+      file: pkg.main,
+      name: 'sankeyCircular',
```

Comment:
oh! so I did try setting `output.name` to `d3` like other d3 modules do it, but when I did that, the d3 object from `d3-circular-sankey` clobbered the main d3 object, and so our example failed to render when it could not find the "main d3" methods it was looking for, like `d3-select`

Reply:
thinks I found the answer

https://github.com/d3/d3-scale/blob/master/rollup.config.js

https://rollupjs.org/guide/en#javascript-api

![screen shot 2018-05-31 at 2 13 22 pm](https://user-images.githubusercontent.com/2119400/40808702-d6c1125c-64dc-11e8-910f-4582bae592a7.png)

Reply:
![screen shot 2018-05-31 at 2 15 39 pm](https://user-images.githubusercontent.com/2119400/40808794-209220ec-64dd-11e8-85ab-65d25a0f251f.png)
https://rollupjs.org/guide/en#advanced-functionality

Reply:
🎉 it's working!  `d3.sankeyCircular` exists and is a function like expected, without any special extra steps for the library user.

@tomshanley this one should be ready to merge now 😄

Reply:
thank you - once I've done the merge, will I need to rebuild and publish to NPM?  i presume so

Reply:
yes, that's right.  if you follow semver, this would be a minor version since we add new functionality (the umd module format at es module format support) without adding any breaking changes to the API ☺

Reply:
excellent.  I'll get on to that this weekend